ossindex/ossindex_test.go

```diff
@@ -0,0 +1,110 @@
+// Copyright 2018 Sonatype Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Definitions and functions for processing the OSS Index Feed
+package ossindex
+
+import (
+	"github.com/sonatype-nexus-community/nancy/types"
+	"github.com/stretchr/testify/assert"
+	"os"
+	"strings"
+	"testing"
+)
+
+func exists(filePath string) (exists bool) {
```

Comment:
NICE!

Reply:
:) Still feels a little odd that this is not part of some standard package, but then again where do you stop if you starting adding such things...